pkg/create/templates/test.go

```diff
@@ -39,7 +39,7 @@ container_ip() {
 }
 
 run_sti_build() {
-  sti build ${sti_args} file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
+  sti build ${sti_args} ${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
```

Comment:
Hey, I see there's no more `git init` step in `prepare()` ... that means that actually using `file://` produce a bad test runner, one that won't work